lib/ContextModule.js

```diff
@@ -645,10 +646,10 @@ webpackEmptyAsyncContext.id = ${JSON.stringify(id)};`;
 		const initialSize = 160;
 
 		// if we dont have dependencies we stop here.
-		return this.dependencies.reduce(
-			(size, dependency) => size + 5 + dependency.userRequest.length,
-			initialSize
-		);
+		return this.dependencies.reduce((size, dependency) => {
+			if (dependency instanceof ModuleDependency)
+				return size + 5 + dependency.userRequest.length;
+		}, initialSize);
```

Comment:
Shouldn't this have a return for the other case?
```
if (dependency instanceof ModuleDependency) {
  return size + 5 + dependency.userRequest.length;
} else {
  return size
}
```
Otherwise `size` will be `undefined` on the next iteration nothing was returned

Reply:
Great catch!!!!!!